sdk/python/feast/data_source.py

```diff
@@ -576,6 +603,7 @@ def __init__(
             created_timestamp_column (optional): Timestamp column when row was created, used for deduplicating rows.
             file_url: [Deprecated] Please see path
             file_format (optional): Explicitly set the file format. Allows Feast to bypass inferring the file format.
+            s3_endpoint_override (optional): Overrides AWS S3 enpoint with internal S3 storage
```

Comment:
Is internal the right word? Or more like custom? The endpoint can be public after all.

Reply:
fixed